src/main/java/uk/gov/di/handlers/AuthorizeHandler.java

```diff
@@ -140,13 +162,36 @@ public Object handle(Request request, Response response) {
             if (formParameters.containsKey("claims-inherited-identity")) {
                 if (!formParameters.get("claims-inherited-identity").trim().isEmpty()) {
                     LOG.info("Inherited Identity record claim requested");
+                    var inheritedIdentity =
```

Comment:
If this block is only valid in integration, can we please pull out a method to do this, and a guard to only action this in the integration environment?

Reply:
Why would it only be valid in integration? I think it should be applicable to staging, and I'm not sure why we'd want to prevent it in other environments